context/app/static/js/components/Header/staticLinks/staticLinks.tsx

```diff
@@ -1,116 +1,277 @@
 import React from 'react';
 
-import LinkGroups, { LinkGroupType, LinksSectionProps } from '../LinkGroups';
+import DeveloperBoardRounded from '@mui/icons-material/DeveloperBoardRounded';
+import PreviewRounded from '@mui/icons-material/PreviewRounded';
+import Button from '@mui/material/Button';
 
-function withLinkGroups(groups: LinkGroupType) {
-  return function LinksSection({ isIndented }: LinksSectionProps) {
-    return <LinkGroups groups={groups} isIndented={isIndented} />;
-  };
-}
+import { type DrawerSection } from 'js/shared-styles/Drawer';
+import {
+  AsctBIcon,
+  EUIIcon,
+  InfoIcon,
+  OrganIcon,
+  SearchIcon,
+  SupportIcon,
+  DonorIcon as UserIcon,
+} from 'js/shared-styles/icons';
+import ExternalImageIcon from 'js/shared-styles/icons/ExternalImageIcon';
+import { entityIconMap } from 'js/shared-styles/icons/entityIconMap';
+import { contactUsUrl } from 'js/shared-styles/Links/ContactUsLink';
+import { DrawerTitle } from 'js/shared-styles/Drawer/styles';
+import AuthButton from '../AuthButton';
 
-const atlasGroups = {
-  ccf: [
-    {
-      href: 'https://humanatlas.io',
-      label: 'Human Reference Atlas (HRA) Portal',
-    },
-    {
-      href: 'https://hubmapconsortium.github.io/ccf-asct-reporter/',
-      label: 'ASCT+B Reporter',
-    },
-    {
-      href: 'https://portal.hubmapconsortium.org/ccf-eui',
-      label: 'Exploration User Interface (EUI)',
-    },
-    {
-      href: 'https://hubmapconsortium.github.io/ccf-ui/rui/',
-      label: 'Registration User Interface (RUI)',
-    },
-  ],
-  azimuth: [
-    {
-      href: 'https://azimuth.hubmapconsortium.org/',
-      label: 'Azimuth: Reference-based Single Cell Mapping',
-    },
-    {
-      href: 'http://fusion.hubmapconsortium.org?utm_source=hubmap',
-      label: 'FUSION: FTU State Identification',
-    },
-  ],
-  hra: [
-    {
-      href: 'https://hubmapconsortium.github.io/hra-previews/pilots/pilot2.html',
-      label: 'HRA Preview: Vasculature CCF Visualization',
-    },
-    {
-      href: 'https://hubmapconsortium.github.io/hra-previews/pilots/pilot3.html',
-      label: 'HRA Preview: HRA vs. Experimental Data',
-    },
-    {
-      href: 'https://hubmapconsortium.github.io/hra-previews/pilots/pilot6.html',
-      label: 'HRA Preview: FTU Segmentation',
-    },
-    {
-      href: 'https://hubmapconsortium.github.io/hra-previews/pilots/pilot7.html',
-      label: 'HRA Preview: Mesh-Level Collision Detection',
-    },
-  ],
-  avr: [
-    {
-      href: 'https://avr.hubmapconsortium.org',
-      label: 'Antibody Validation Reports',
-    },
-  ],
-};
+export const resourceLinks: DrawerSection[] = [
+  {
+    title: 'User Guides',
+    items: [
+      {
+        label: 'Tutorials',
+        description: 'Learn more about how to explore the data portal',
```

Comment:
Do we want punctuation on these descriptions or is that handled elsewhere?

Reply:
The original mockups don't seem to have descriptions: 
![image](https://github.com/hubmapconsortium/portal-ui/assets/19957804/e75fffba-5cd3-4706-bf7b-6264deaf82c8)